sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionContext.cs

```diff
@@ -29,21 +32,76 @@ public class PartitionContext
         ///
         public string PartitionId { get; }
 
+        /// <summary>
+        ///   The identifier of the associated <see cref="EventProcessor{T}" /> instance.
+        /// </summary>
+        ///
+        public string OwnerIdentifier { get; }
+
+        /// <summary>
+        ///   Interacts with the storage system, dealing with the creation of checkpoints.
+        /// </summary>
+        ///
+        private PartitionManager Manager { get; }
+
         /// <summary>
         ///   Initializes a new instance of the <see cref="PartitionContext"/> class.
         /// </summary>
         ///
         /// <param name="eventHubName">The name of the specific Event Hub this context is associated with, relative to the Event Hubs namespace that contains it.</param>
         /// <param name="consumerGroup">The name of the consumer group this context is associated with.</param>
         /// <param name="partitionId">The identifier of the Event Hub partition this context is associated with.</param>
+        /// <param name="partitionManager">Interacts with the storage system, dealing with the creation of checkpoints.</param>
```

Comment:
We may want to consider tuning the language here a bit.  Maybe something like:

`Interacts with the storage system with responsibility for creation of checkpoints`